third_party/terraform/resources/resource_storage_bucket.go

```diff
@@ -471,6 +491,11 @@ func resourceStorageBucketRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("versioning", flattenBucketVersioning(res.Versioning))
 	d.Set("lifecycle_rule", flattenBucketLifecycle(res.Lifecycle))
 	d.Set("labels", res.Labels)
+
+	if res.Billing != nil {
```

Comment:
I think we should also have an else on this to d.Set to nil so that if the field gets removed out of band, it's removed from state.

Reply:
That could have been a flatten, but for now I'm just doing it inline.